File: Bubble-Sort/python/leetcode_lc_283.py
nums = [0,1,0,3,12]


class Solution:
    def moveZeroes(self, nums) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """

        for i in range(0,len(nums)):
            for j in range(0,len(nums)-i):
                if(nums[j] == 0):
                    nums.remove(nums[j])
                    nums.append(0)
                    break


solution = Solution()

solution.moveZeroes(nums)

print(nums)

File: Bubble-Sort/js/leetcode_lc_283.js
const arr = [0,1,0,3,12]

/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var moveZeroes = function(nums) {
    for(let i = 0 ; i < nums.length ; i++){
        for(let j = 0 ; j < nums.length - i ; j++){
            if(nums[j] === 0){
                nums.splice(j,1)
                nums.push(0)
                break
            }
        }
    }
    return nums
};

console.log(moveZeroes(arr))